.github/actions/spelling/reject.txt

```diff
@@ -0,0 +1,22 @@
+^attache$
```

Comment:
These are the words that were removed from dictionary.txt. (It's possible to compress this list as it's composed of regular expressions, but there's no particular need.)